### PR TITLE
Add SPCON tool which lets you not to compile Sprout8 ROM

### PR DESCRIPTION
**Pull request by saikrishna279**
This tools lets you make Sprout8 ROM instead converts the Sprout4 ROM to Sprout8 ROM which lets you not to waste time on Sprout8 builds...

Commands:
spcon
cleansp

### CODE REVIEW

File: 48c/make.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
+blue='\033[0;34m'
+cyan='\033[0;36m'
+yellow='\033[0;33m'
+red='\033[0;31m'
+nocol='\033[0m'
+cd out/target/product/sprout4/
```
Comment (akhilnarang) [inline 48c/make.sh:6]: you're just assuming builder using default out dir

**Comment by akhilnarang** [comment]
Just flash a sprout8 boot.img, no need of all this :v 
@Bun-Bun69 anyway has no issue with compiling both.

**Comment by Shreeesha** [comment]
Well, no issue making builds for two variants

**Comment by saikrishna279** [comment]
But, different ROM's use diferent boot.img, so an universal boot.img wont work Akhil

**Comment by akhilnarang** [comment]
@saikrishna279 which roms use a different one

**Comment by saikrishna279** [comment]
@akhilnarang  For example, AICP has a rc file called init.aicp.rc, tesla has init.tesla.rc and they change by ROM, just copying a single boot.img will give you boot loop and I've tried that.

**Comment by akhilnarang** [comment]
Make an anykernel zip to swap out just teh needed parts of teh ramdisk then

**Comment by saikrishna279** [comment]
I would if i could execute a pearl script on recovery! XD

**Comment by akhilnarang** [comment]
why do you need pearl :|
take thunderzap zip
just change the replacing zImage part to replace init and fstab instead

**Comment by saikrishna279** [comment]
Wut wut wut wut wut wut wut!!!! boot.img has two parts when you extract it---
1)zImage
2)ramdisk.gz
The init.sprout.rc and fstab.sprout are in ramsdik.gz and you have to extract it for conversion prupose...
You cannot extract it on recovery like a zip, AND THAT'S WHY THE PEARL SCRIPT IS FOR!
You also need the alloc size for recompression which is what you cant do in a Recovery...

**Comment by akhilnarang** [comment]
Ohk
The boot.img extractor I used directly unpacked the whole boot.img

**Comment by saikrishna279** [comment]
In recovery?

**Comment by akhilnarang** [comment]
Nop
Cmd

**Comment by saikrishna279** [comment]
Well, that's what it is for, you directly convert it instead of CMD or another build

**Pull request closed without merging**